src/wallet/wallet.h

```diff
@@ -651,6 +608,8 @@ class CWalletCert : public CScCertificate, public CWalletTransactionBase
         READWRITE(hashBlock);
         READWRITE(vMerkleBranch);
         READWRITE(nIndex);
+        READWRITE(bwtMaturityDepth);
+        READWRITE(areBwtCeased);
         std::vector<CScCertificate> vUnused; //! Used to be vtxPrev
```

Comment:
Unused fields could be probably removed from this cert obj because it is a brand new data type in wallet db

Reply:
Agree, since CWalletCert is stored under different key than CWalletTx